packages/gguf/src/gguf.ts

```diff
@@ -413,7 +417,7 @@ export async function ggufAllShards(
 			urls.push(`${prefix}-${shardIdx.toString().padStart(5, "0")}-of-${total.toString().padStart(5, "0")}.gguf`);
 		}
 
-		const PARALLEL_DOWNLOADS = 20;
+		const PARALLEL_DOWNLOADS = params?.parallelDownloads ?? 20;
```

Comment:
Nitpicks: This variable should no longer be all caps, since it's no longer a constant.

My suggestion is to remove this line and replace line 423 with `parallelDownloads`